### ISSUE TITLE
logo.svg missing

### ISSUE THREAD

**Issue by CharlesChiuGit** [ahonn/logseq-plugin-todo#6]
In the console:
```
GET file:///C:/Users/{username}/.logseq/plugins/logseq-todo-plugin/logo.svg net::ERR_FILE_NOT_FOUND
```
but it's in your repo, which is kinda odd. Maybe something goes wrong while building it?

**Comment by CharlesChiuGit** [comment]
fixed in v1.2.1

**Comment by ahonn** [comment]
Thanks for the feedback, the problem was a missing logo.svg when packing into zip after build. it's fixed now.
